Code/Mantid/Framework/DataHandling/src/LoadNexusLogs.cpp

```diff
@@ -173,7 +173,9 @@ void LoadNexusLogs::exec() {
       std::vector<Mantid::Kernel::DateAndTime> ptime;
       pval.reserve(event_frame_number.size());
       ptime.reserve(event_frame_number.size());
-      std::vector<Mantid::Kernel::DateAndTime> plogt = plog->timesAsVector();
+      std::vector<Mantid::Kernel::DateAndTime> plogt;
+      if (plog)
+        plogt = plog->timesAsVector();
       std::vector<double> plogv = plog->valuesAsVector();
```

Comment:
This also has the aforementioned issued - a second dereference that's unchecked.